README.md

```diff
@@ -18,7 +18,7 @@ This demo uses the data from the hosted Metabase Cloud instance and provides a h
 
 - Run the React frontend. This starts the demo application on HTTPS, which is required to connect to the hosted JWT server.
 
-  - `yarn start:https`
+  - `yarn dev:https`
```

Comment:
Interestingly, I've always been running `yarn start` and `yarn dev` in this PR with hosted JWT, and it still seems to work for me. Can you double-check if this is really a requirement? I mean having the HTTPS option is great in case we really need to test features that require HTTPS, but I don't want people to get the wrong idea that they only need to use this command when using hosted JWT server.

Reply:
@WiNloSt The `dev:https` is required if you want to use our hosted API server on Vercel, without running up the API server locally (which is not possible if you are not a Metabase employee, as you need the `DB_URL`)

I decided to make that option the most prominent, as that is the only way non-employees can run the demo. In the below sections, the instruction says that Metabase employees can run locally with `yarn dev` and `cd api && yarn dev`

Reply:
@WiNloSt woah, I just tried `yarn dev` without HTTPS and it now works too! I swear that never worked back when I used create-react-app, not sure what changed...

In any case, I'll update README accordingly. Thanks!